book/src/guide/run.md

```diff
@@ -34,6 +47,24 @@ To do so add the following to `Cargo.toml`:
 enabled = true
 ```
 
+### Refinement Annotations on a Package
+
+Adding refinement annotations to packages is simple. You can add `flux-rs` as a dependency in `Cargo.toml`
+
+```toml
+[dependencies]
+flux-rs = { git  = "https://github.com/flux-rs/flux.git" }
+```
+
+Then, add refinement annoations using the fully qualified path `flux_rs::`.
```

Comment:
I've been tending towards importing all the attributes with `flux_rs::*` and then using them unqualified like so

```rust
use flux_rs::*;

#[sig(fn(x: i32) -> i32{v: x < v)]
fn inc(x: i32) -> i32 {
    x - 1
}
```

It is a matter of taste, but maybe we can suggest that as an alternative.